content/src/content/jcr_root/apps/core/wcm/sandbox/components/carousel/v1/carousel/README.md

```diff
@@ -43,7 +43,7 @@ BLOCK cmp-carousel
     ELEMENT cmp-carousel__content
     ELEMENT cmp-carousel__item
     ELEMENT cmp-carousel__action
-        MOD cmp-carousel__action--prev
+        MOD cmp-carousel__action--previous
```

Comment:
we use full words for all other hooks and JS variables elsewhere, so it makes sense to do the same here.